validation-test/BuildSystem/skip-local-build.test-sh

```diff
@@ -1,8 +1,12 @@
 # REQUIRES: standalone_build
-#
-# RUN: %swift_src_root/utils/build-script --dump-config --skip-local-build 2>&1 | %FileCheck %s -check-prefix=CONFIG
+
+# RUN: %empty-directory(%t)
+# RUN: mkdir -p %t
```

Comment:
`%empty-directory(%t)` expands to `rm -rf %t && mkdir -p %t` so this is redundant.

Reply:
Thanks for noticing this!

Tackled in 7379f455bf06d3e29ed895256f3b622fde0d50bc -- took the chance to rebase the PR so that there is a separate commit to reenable `lvm-targets-options.test`